### PR TITLE
clean up IVLOG1 logs

### PR DESCRIPTION
**Pull request by adstraw**

### CODE REVIEW

File: pmlc/dialect/pxa/transforms/vectorize.cc [hunk 1]
```diff
@@ -305,15 +305,16 @@ LogicalResult vectorizeOverOutputs(AffineParallelOp op, unsigned vectorWidth) {
   }
   auto reduce = dyn_cast<PxaReduceOp>(getPrevWriter(op.getResult(0)));
   if (!reduce) {
-    return op.emitRemark(
-        "vectorizeOverOutputs: Failed, missing previous PxaReduceOp");
+    return failure();
```
Comment (adstraw) [inline pmlc/dialect/pxa/transforms/vectorize.cc:308]: We are hitting this case a LOT on ResNet50 which bulks up the logs quite a bit as we print the entire op IR when we do a remark.  Not sure if this is the right thing to do here, or not.

**Review by jbruestle — APPROVED** [review]

File: pmlc/dialect/pxa/transforms/vectorize.cc [hunk 2]
```diff
@@ -305,15 +305,16 @@ LogicalResult vectorizeOverOutputs(AffineParallelOp op, unsigned vectorWidth) {
...
+    // return op.emitRemark(
+    //    "vectorizeOverOutputs: Failed, missing previous PxaReduceOp");
```
Comment (flaub) [inline pmlc/dialect/pxa/transforms/vectorize.cc:310]: Remove comments

File: plaidml/op/lib/ops.cc [hunk 3]
```diff
@@ -431,7 +431,7 @@ std::vector<int64_t>* extend_manual_padding(std::vector<int64_t>* pads, size_t r
 }  // namespace
 
 Value abs(const Value& value) {
-  IVLOG(1, "abs");
+  IVLOG(2, "abs");
```
Comment (flaub) [inline plaidml/op/lib/ops.cc:434]: Hmm, I think these are good to have at this level.

File: pmlc/dialect/tile/builder.cc [hunk 4]
```diff
@@ -578,7 +578,7 @@ TileBuilder::MakeProgram(StringRef name, const ProgramMutations &mutations,
   if (name.empty()) {
     name = "noname";
   }
-  IVLOG(1, "TileBuilder::MakeProgram> " << name.str());
+  IVLOG(2, "TileBuilder::MakeProgram> " << name.str());
```
Comment (flaub) [inline pmlc/dialect/tile/builder.cc:581]: I was hoping to keep this one at 1

File: plaidml/op/lib/ops.cc (hunk 3 above)
Comment (jbruestle) [reply plaidml/op/lib/ops.cc:434]: Why specifically?  I'd like to keep log level 1 to basic status, basically something with O(1) lines per program execution or thereabouts.  Basically if you run resnet50 at log level 1, it spews > 2k lines prior to this commit.

Comment (flaub) [reply plaidml/op/lib/ops.cc:434]: Yeah, this should be happening at the O(1) level.

File: pmlc/dialect/tile/builder.cc (hunk 4 above)
Comment (jbruestle) [reply pmlc/dialect/tile/builder.cc:581]: Does this get called every time Keras does shape inference?  I think that might be the issue.  If not, I agree that it's high level enough to keep.

Comment (adstraw) [reply pmlc/dialect/tile/builder.cc:581]: As an example, this is called 342 times during a ResNet50 run on Intel Gen9 GPU.  Each time we print quite a lot of IR.  Log size takes a big hit:

```
$ ls -la ~/temp/logMakeProgram 
-rw-rw-r-- 1 adstraw adstraw 37026716 Sep 14 13:03 <path>/logMakeProgram

$ ls -la ~/temp/noLogMakeProgram 
-rw-rw-r-- 1 adstraw adstraw 196395 Sep 14 13:09 <path>/noLogMakeProgram
```

File: pmlc/dialect/pxa/transforms/vectorize.cc (hunk 2 above)
Comment (adstraw) [reply pmlc/dialect/pxa/transforms/vectorize.cc:310]: Will do.

File: plaidml/op/lib/ops.cc (hunk 3 above)
Comment (adstraw) [reply plaidml/op/lib/ops.cc:434]: What's the verdict?  O(1) or O(2)?  These logs aren't very illuminating, in my opinion.  How does logging the operation name help?

File: pmlc/dialect/tile/builder.cc (hunk 4 above)
Comment (flaub) [reply pmlc/dialect/tile/builder.cc:581]: OK, yes, I could see this causing lots of logs for keras. Level 2 is fine here.

File: plaidml/op/lib/ops.cc (hunk 3 above)
Comment (flaub) [reply plaidml/op/lib/ops.cc:434]: Usually, this comes up like something crashed and it'd be interesting to see which operation caused it. The `O(1)` means that we expect the log to be printed once (not like `O(n)` or `O(n*n)`). Overall it's not a big deal, I just didn't think the oplib logs were excessive so far.

Comment (adstraw) [reply plaidml/op/lib/ops.cc:434]: OK, I think it is reasonable to put these back.  Slight increase in log size for ResNet50.  235535 vs. 196395.  Same order of magnitude.

**Comment by adstraw** [comment]
Follow up PR:
https://github.com/plaidml/plaidml/pull/1357

**Pull request merged**